### PR TITLE
WW-93: Update uncyc footer for ComScore tracking

### PR DESCRIPTION
**Pull request by kerriamber**
@Wikia/west-wing 

Ticket: https://wikia-inc.atlassian.net/browse/WW-93

### CODE REVIEW

File: extensions/GlobalMessages/GlobalMessagesV.i18n.php [hunk 1]
```diff
@@ -237,6 +238,7 @@
 	'vertical-lifestyle' => 'Lifestyle vertical name',
 	'vertical-music' => 'Music vertical name',
 	'vertical-movies' => 'Movies vertical name',
+	'vertical-comedy' => 'Uncyclopedia wikis comscore tag label',
```
Comment (sebastianmarzjan) [inline extensions/GlobalMessages/GlobalMessagesV.i18n.php]: I suggest to use the convention above, with a small addition -
'Comedy vertical name, used only by Uncyclopedia'

File: skins/uncyclopedia/main.css [hunk 2]
```diff
@@ -1845,3 +1847,42 @@ p.mw-ipb-conveniencelinks {
...
+
+#uncyc-branding .wikia-logo span {
+	display: inline-block;
+	float: left;
+	font-weight: bold;
+	height: 40px;
+	line-height: 42px;
+	margin: 0 20px;
+	text-transform: uppercase;
+}
```
Comment (sebastianmarzjan) [inline skins/uncyclopedia/main.css]: missing NL

File: skins/uncyclopedia/main.css [hunk 3]
```diff
@@ -1095,6 +1095,8 @@ input.searchButton {
 #f-copyrightico {
 	float: left;
 	height: 1%;
+	width: 88px;
+	background-repeat: no-repeat;
```
Comment (sebastianmarzjan) [inline skins/uncyclopedia/main.css]: The conventions require to sort properties alphabetically

File: skins/monobook/main.css [hunk 4]
```diff
@@ -1238,3 +1244,43 @@ div.mw-lag-warn-high {
 
 
 /* end Wikia monobook modification */
+
```
Comment (sebastianmarzjan) [inline skins/monobook/main.css:1247]: Please use a SASS mixin for this element to avoid C&P in two files, unless not possible

Comment (kerriamber) [inline skins/monobook/main.css:1247]: @SebastianMarzjan Unfortunately, I don't think I can use a SASS mixin here.

Comment (sebastianmarzjan) [inline skins/monobook/main.css:1247]: Right, those are pure CSS files 🙈

**Comment by sebastianmarzjan** [comment]
LGTM

**Pull request merged**